tests/scripts/start-storybook.sh

```diff
@@ -1,21 +1,28 @@
 #!/bin/bash
 
-npm run storybook -- --quiet --ci &
-JOB_ID=$(echo $!) #get the background job ID
+#check for existing process on port:6006
+process=`lsof -i :6006`
+if [[ -z $process ]]
```

Comment:
This is probably a dumb question but what is the `-z` here?

Reply:
This is to check if the variable is undefined, and returns true here if there is nothing returned with the `lsof -i :6006` command.